libcrux-ml-dsa/src/simd/avx2/arithmetic.rs

```diff
@@ -3,9 +3,9 @@ use crate::{
     simd::traits::{FIELD_MODULUS, INVERSE_OF_MODULUS_MOD_MONTGOMERY_R},
 };
 
-use libcrux_intrinsics::avx2::*;
+use libcrux_intrinsics::avx2::{mm256_setzero_si256, *};
```

Comment:
```suggestion
use libcrux_intrinsics::avx2::*;
```
This is redundant, no?